src/routes/paths/SingleRightPath.tsx

```diff
@@ -0,0 +1,4 @@
+export enum SingleRightPath {
+  OfferedSingleRights = 'offered-single-rights',
```

Comment:
Should this be offered? Perhaps DelegateSingleRights is better?
```suggestion
  DelegateSingleRights = 'delegate-single-rights',
```